insilico_panels/KK/MAD_och_MAD-liknande_defekter.v1.0/MAD_och_MAD-liknande_defekter.v1.0.bed

```diff
@@ -5,5 +5,5 @@
 8	145582216	145584946	SLC52A2	+
 8	104410865	104427563	SLC25A32	-
 9	79000432	79009444	RFK	-
-15      76508627        76603810        ETFA    0       -
-19      51848408        51869672        ETFB    0       -
+15	76508627	76603810	ETFA	0	-
+19	51848408	51869672	ETFB	0	-

```

Comment:
Was this lost during my git history cleanup?

Reply:
no that was in wopr, it should be ok there